### ISSUE TITLE
the loss is nan

### ISSUE THREAD

**Issue by bbbeach** [yjh0410/PyTorch_YOLOv3#1]
I train the yolov3 model with voc2007 and voc 2012.In the second epoch,the loss turn up nan
